static/docs/command-reference/repro.md

```diff
@@ -44,6 +44,10 @@ files, intermediate or final results. It saves all the data files, intermediate
 or final results into the <abbr>DVC cache</abbr> (unless `--no-commit` option is
 specified), and updates stage files with the new checksum information.
 
+### Locking
```

Comment:
The `locking` itself does not probably mean anything to users. Even from a technical perspective - locking is a solution to ... <what?>. What would the user level problem they are facing? Running multiple repros in parallel? Interested in parallelism? Guarantees about parallel mode? etc...